jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/RequestLogger.java

```diff
@@ -35,10 +42,43 @@ public static void headersToString(HttpServerRequest request, StringBuilder logL
      */
     public static void headersToString(HttpServerResponse response, StringBuilder logLine) {
         response.responseHeaders().forEach(header ->
-            logLine.append(header.getKey()).append('=').append(header.getValue()).append(' ')
+            logLine.append(header.getKey())
+                .append('=')
+                .append(getHeaderValueOrRedact(header))
+                .append(' ')
         );
     }
 
+    /**
+     * Returns the value of a header. If the header contains sensitive information the value will be replaced with "REDACTED".
+     *
+     * @param header The header to get the value from
+     * @return value of the header or "REDACTED" if the header contains sensitive information
+     */
+    public static String getHeaderValueOrRedact(Map.Entry<String, String> header) {
+        if ("Authorization".equalsIgnoreCase(header.getKey())) {
```

Comment:
A small optimization would be to add "Authorization" and "Redacted" strings att static strings

Reply:
From a performance point of view, I don't think it matters

Reply:
Will the compiler turn it into a static final? Otherwise the strings are freshly created each time and we will gain a performance boost from making it static final.

But not many calls come through here so it makes small difference.